src/lsm/forest_fuzz.zig

```diff
@@ -748,11 +758,22 @@ const Environment = struct {
 
                 // Asserting the positive space:
                 // all objects found by the scan must exist in our model.
-                for (accounts) |account| {
+                for (accounts) |*account| {
                     const prefix_current: u128 = switch (params.index) {
-                        inline else => |field| @field(account, @tagName(field)),
+                        .imported => index: {
+                            assert(params.min == 0 and params.max == 0);
+                            assert(account.flags.imported);
+                            break :index 0;
```

Comment:
I'll take a look, but it's because the fuzzer uses a `u128` as a type-erased prefix (so it's no aware of the field's type).
In fact, not mixing zero with "nothing" was the main reason I chose to use `void` instead of `u0` for the `CompositeKey` prefix.